src/main/resources/application.properties

```diff
@@ -225,3 +225,6 @@ spring.flyway.ignore-missing-migrations=true
 spring.flyway.baseline-on-migrate=true
 spring.flyway.baseline-version=0001
 
+reference.data.api.url=${RD_LOCATION_REF_API_BASE_URL:http://localhost:5555}
+reference.data.cache.ttl.in.days=5
+reference.data.cache.refresh.rate.in.milliseconds=86400000
```

Comment:
Would it be clearer to specify this in hours ? Do we need millisecond granularity ?

Reply:
changing to use `cron`: refresh everyday at 4am